app/assets/javascripts/civil_applications/civil_applications.js

```diff
@@ -0,0 +1,57 @@
+var civil_applications = {
```

Comment:
**[not a blocker]** 📝 These files seem specific to the `citizen` frontend (they'll be added later anyway), not to the setup of the app to support the asset pipeline.

Reply:
To be honest - I copied the stuff across from the other PR and removed the provider stuff ... well I thought I had.